ui/packages/catalog-ui-search/src/main/webapp/component/property/property.less

```diff
@@ -81,6 +88,25 @@
     .background-gradient(rgba(0,0,0,.05), rgba(255,255,255,.05));
     z-index: 0;
   }
+
+  .for-error {
```

Comment:
❗️ This can be simplified to:
```
.for-error {
    color: contrast(@negative-color);
    display: none;
    line-height: @minimumLineSize;
    padding-left: @minimumSpacing;
    background: @negative-color;
    height: auto;
  }

  &.failed-required-check.show-required-warning {
    .for-error {
      display: block;
    }
  }
```